File: unit11-hw/ruby/Kiwi.rb
# Kiwi.rb | Defines the Kiwi class which inherits attributes and methods
#   from the Bird superclass.
#
# Begun by: Dr. Jump for CS 315 at King's College.
# Completed by: Robert Lowman
# Date: 11/29/16
####################################################

require_relative 'WalkingBird.rb'

class Kiwi < WalkingBird

  def call
    'Twee-Do'
  end

end

File: unit11-hw/python/Kiwi.py
# Kiwi.py | Defines the Kiwi class which inherits attributes and methods
#   from the Bird superclass.
#
# Completed by: Robert Lowman
# Date: 12/6/16
####################################################
import WalkingBird

class Kiwi(WalkingBird.WalkingBird):

    def __init__(self, name):
        WalkingBird.__init__(name)
        self.name = name
        
    def call(self):
        return 'Twee-do!'

    def className(self):
        return 'Kiwi'
